tools/container/crio/crio.go

```diff
@@ -220,8 +221,15 @@ func setupHook(o *options) error {
 func setupConfig(o *options) error {
 	log.Infof("Updating config file")
 
+	var cliArgs []string
+	if o.HostRootMount != "" {
+		cliArgs = append(cliArgs, "chroot", o.HostRootMount)
+	}
+	cliArgs = append(cliArgs, "crio", "status", "config")
+
```

Comment:
Seems like this should be in a function.

Reply:
Done